src/java.base/share/classes/com/sun/crypto/provider/PrivateKeyInfo.java

```diff
@@ -87,4 +93,8 @@
     AlgorithmId getAlgorithm() {
         return this.algid;
     }
+
+    public void clear() {
+        Arrays.fill(privkey, (byte)0);
```

Comment:
check for null just in case?

Reply:
`val.data.getOctetString()` on line 82 should never return null, so I think a null check is not necessary here.

Reply:
Sure.